<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameScreen</class>
 <widget class="QDialog" name="GameScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="gameScreenPicLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inventoryB">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>642</y>
     <width>193</width>
     <height>54</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gameScreenSettingsB">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>120</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="clockLabel">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="informationTB">
   <property name="geometry">
    <rect>
     <x>823</x>
     <y>153</y>
     <width>423</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="eraseB">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>120</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mapOneB">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>101</y>
     <width>29</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mapReturnB">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>642</y>
     <width>193</width>
     <height>54</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leftB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightB">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>520</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mapOneFishB">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>380</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mapOneShopB">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="straightB">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backB">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>500</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="battleB">
   <property name="geometry">
    <rect>
     <x>513</x>
     <y>642</y>
     <width>193</width>
     <height>54</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
